launch:
  ros__parameters:
    joy: true
    image_view: false
    rviz: false

controller:
  ros__parameters:
    linear_max_vel: 5.0
    angular_max_vel: 1.57
     
data_collector:
  ros__parameters:
    image_topic: "/image_raw"
    cmd_vel_topic: "/cmd_vel"
    image_width: 480
    image_height: 300
    output_path: "dataset.pt"
    max_data_count: 10000